src/timezonepicker/timezone-picker.js

```diff
@@ -73,10 +73,13 @@ class TimezonePicker extends React.Component<
           }
         }
 
+        const offsetMinutes = offset * 60;
+
         timezones.push({
           id: zoneName,
           label,
-          offset,
+          // offset output is in minutes, difference of UTC and this zone (negative for hours ahead of UTC, positive for hours behind)
+          offset: offsetMinutes === 0 ? 0 : offsetMinutes * -1,
```

Comment:
would `offset: offsetMinutes * -1` work instead?

Reply:
offsetMinutes* -1 returns -0 for 0 which isn’t accurate or what we would want to return in this case.